src/software/ai/passing/pass.cpp

```diff
@@ -19,49 +15,35 @@ Point Pass::receiverPoint() const
     return receiver_point;
 }
 
-Angle Pass::receiverOrientation() const
+Angle Pass::receiverOrientation(const Point& ball_position) const
 {
-    return (passerPoint() - receiverPoint()).orientation();
+    return (ball_position - receiverPoint()).orientation();
```

Comment:
Similar idea here, I'd suggest keeping this as the passer point to allow for cases in the future where the passer point is not the current ball position.

Reply:
added back